### ISSUE TITLE
Open leader's guide to more open performance reviews

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization-ambassadors/editorial#11]
I know @arob98 has a keen interest in this subject, so I'm tagging her on it! Maybe one or two writers want to tag-team an article on opening up the performance review process a bit?

**Comment by shabnoorshah** [comment]
Hi I'm new here...but would be happy to participate in this effort if this is still open.

**Comment by semioticrobotic** [comment]
Indeed, it is, @shabnoorshah! I know this would be a valuable resource for the community. We can continue to discuss your plans here, in-thread, or you're welcome to email me to review strategy, outlines, drafts—whatever you like.

**Comment by semioticrobotic** [comment]
Just checking on the status of this, @shabnoorshah. Any update on progress? Most importantly: How can I help?

**Comment by LauraHilliger** [comment]
From the 2021 strategy session:
Potential content - how to handle performance issues on your team as a manager when following the open management practices / open leadership (I believe @amatlack was thinking about this?)

**Comment by shabnoorshah** [comment]
Hi Everyone, just realised I missed some conversations here from before. Apologies @semioticrobotic, messages here were totally off my radar. This is the first time I got a notification from this group:).  Considering we are in touch off-line, I'm hoping I didn't miss much. 

@Laura, Regarding the new topic "how to handle performance issues on your team as a manager when following the open management practices / open leadership", I'm happy to pair or mob to create some useful content/blog/ resource. Let me know.

**Comment by LauraHilliger** [comment]
Great @shabnoorshah! If you would like to start writing or if you already have a draft post on this topic, post a link (to a google doc maybe?) and I'm sure we can get some collaboration happening!

**Comment by amatlack** [comment]
Hey folks, my contribution to planning yesterday was to write something on
handling performance *problems* (not just giving reviews in general) using
open principles, but happy to bow out if someone else wants to take the
topic.

**Comment by semioticrobotic** [comment]
Thanks for the clarification, @amatlack. It sounds to me like we've got two angles on "performance" that we can therefore address: "performance issues" and "performance reviews." Maybe we make these two articles in the same series, then do a video interview/roundtable to summarize and explore?

**Comment by shabnoorshah** [comment]
@LauraHilliger Thanks! What's the time-frame we are looking at. I'll need to update my backlog and prioritize accordingly. 

@amatlack I'm happy to pair on this if you like (but no pressure).

**Comment by amatlack** [comment]
Go ahead if you've got the time. I won't be able to get to it for a while
anyway. :)

**Comment by amatlack** [comment]
@shabnoorshah is this something you're interested in taking on, still? if not, we can probably close it out just to clean house a bit

**Comment by semioticrobotic** [comment]
I'd still love for us to publish something like this!

**Comment by shabnoorshah** [comment]
Hi Allison, Thanks for checking in. While it's a topic I want to explore
but I currently have my hands full. Therefore can't commit as of now. :(

(*My Working Hours May Not be Your working Hours. Please Do Not Feel
Obligated To Reply Outside of Your Normal Work Schedule*)

Shabnoor Shah

She / Her / Hers

Principal Open Leadership Global Lead & Coach, (PCC ICF
<https://www.credly.com/badges/a3809133-6890-4bda-9004-903e1a41fd4c/public_url>
)

Open Innovation Labs

Red Hat GmbH <https://www.redhat.com/> 85630 Grasbrunn Germany

***@***.*** / M: +49-160-9644-2635  /
<https://www.redhat.com/>

- Open leadership
<https://theopenorganization.org/definition/open-leadership-definition/>
Definition

*- Join the open leadership CoP on The Source
<https://source.redhat.com/communitiesatredhat/communitiesofpractice/cross_cutting_co/open_leadership_cop>*

- Hear my views about Open and personal leadership at Leaders Revealed
<https://source.redhat.com/groups/public/podcast_leaders_revealed/leaders_revealed_episodes/episode_22_shabnoor_shah>
podcast

- New E-book : Build a Resilient IT Culture
<https://enterprisersproject.com/article/2023/1/ebook-build-resilient-it-culture>


__________________________________________________________________

Red Hat GmbH <https://www.redhat.com/de/global/dach>, Registered seat:
Werner von Siemens Ring 14, D-85630 Grasbrunn, Germany
Commercial register: Amtsgericht Muenchen/Munich, HRB 153243,
Managing Directors: Ryan Barnhart, Charles Cachera, Michael O'Neill, Amy Ross

**Comment by shabnoorshah** [comment]
Hi Bryan,
I agree it's a meaningful topic. Maybe we can pick it up at later date?

(*My Working Hours May Not be Your working Hours. Please Do Not Feel
Obligated To Reply Outside of Your Normal Work Schedule*)

Shabnoor Shah

She / Her / Hers

Principal Open Leadership Global Lead & Coach, (PCC ICF
<https://www.credly.com/badges/a3809133-6890-4bda-9004-903e1a41fd4c/public_url>
)

Open Innovation Labs

Red Hat GmbH <https://www.redhat.com/> 85630 Grasbrunn Germany

***@***.*** / M: +49-160-9644-2635  /
<https://www.redhat.com/>

- Open leadership
<https://theopenorganization.org/definition/open-leadership-definition/>
Definition

*- Join the open leadership CoP on The Source
<https://source.redhat.com/communitiesatredhat/communitiesofpractice/cross_cutting_co/open_leadership_cop>*

- Hear my views about Open and personal leadership at Leaders Revealed
<https://source.redhat.com/groups/public/podcast_leaders_revealed/leaders_revealed_episodes/episode_22_shabnoor_shah>
podcast

- New E-book : Build a Resilient IT Culture
<https://enterprisersproject.com/article/2023/1/ebook-build-resilient-it-culture>


__________________________________________________________________

Red Hat GmbH <https://www.redhat.com/de/global/dach>, Registered seat:
Werner von Siemens Ring 14, D-85630 Grasbrunn, Germany
Commercial register: Amtsgericht Muenchen/Munich, HRB 153243,
Managing Directors: Ryan Barnhart, Charles Cachera, Michael O'Neill, Amy Ross

**Comment by ldimaggi** [comment]
Would anyone like to collaborate on writing this article? Thx!
